### DISCUSSION TITLE
Recent regression in Network Mode?

### DISCUSSION THREAD

**Discussion by matthewhuang97** [TanStack/query#5631] [Q&A]
Today, we received a couple of bug reports that data wasn't loading in our product, and we narrowed it down to Network Mode incorrectly reading the state of the agent as offline, and therefore not firing the network requests.

This was strange because nothing seems to have changed recently to cause this, and it's a really rare bug. I'm wondering if other folks in the community have run into this too? Could it be related to the new Chrome update that was released today?

**Comment by TkDodo** [comment]
This used to be an issue on macs with chrome, where `navigator.onLine`, which we use to determine the offline status, has falsy reported `false` even though there was network connection.

there are a bunch of bugs reported: https://bugs.chromium.org/p/chromium/issues/list?q=navigator.online

fwiw, I also updated today to Chromium 114.0.5735.133 (I'm using brave though), and it still works fine for me.
